Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+
+
+RUN apt-get update && apt-get install -y curl apt-transport-https
```

Comment:
I think it's better to combine the next 6 lines into one `RUN` instruction. 
Take a look at https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run